buildscripts/test.py

```diff
@@ -234,20 +237,26 @@
         total_passed = len(passed_examples)
         total_failed = len(failed_examples)
         total_expected_failures = len(expected_failures)
-        total_run = total_passed + total_failed + total_expected_failures
+        total_unexpected_success = len(unexpected_success)
+        total_run = total_passed + total_failed + total_expected_failures + total_unexpected_success
         format_print(' '.join([f'SDC examples summary:',
                                f'{total_run} RUN,',
                                f'{total_passed} PASSED,',
                                f'{total_failed} FAILED,',
-                               f'{total_expected_failures} EXPECTED FAILURES']))
+                               f'{total_expected_failures} EXPECTED FAILURES',
+                               f'{total_unexpected_success} UNEXPECTED SUCCESS',
+                               ]))
         for item in passed_examples:
             format_print(f' - {item}: PASSED', new_block=False)
         for item in failed_examples:
             format_print(f' - {item}: FAILED', new_block=False)
         for item in expected_failures:
             format_print(f' - {item}: EXPECTED FAILED', new_block=False)
+        for item in unexpected_success:
+            format_print(f' - {item}: UNEXPECTED SUCCESS', new_block=False)
 
-        sys.exit(0 if total_failed == 0 else -1)
+        success = (total_failed + total_unexpected_success) == 0
+        sys.exit(0 if success else -1)
```

Comment:
Do we really need to fail build if some "expected failed" example passed?
I want to suggest to just print this case.

Reply:
Failed build will attract attention that we should remove example from `expected_failure_list`. Or we will face situation when it become OK, than after some time become FAIL and we will not catch this jumps.